runtimes/php-8.0/prepare.php

```diff
@@ -0,0 +1,4 @@
+<?php
```

Comment:
What is this file used for?

Reply:
Similar to previous comment. If there is composer.json, it MUST have name `user/function`. So this script makes sure user's composer.json is updated with correct name.

Reply:
It will automatically rename the "name" field of the user's composer.json to "user/function" as that is required in the server's composer file to automatically install the user's dependencies. This file makes it so the user themselves does not have to do it.